src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -118,8 +119,9 @@ public async Task<int> DeployAsync()
                 azureSubscriptionClient = azureClient.WithSubscription(configuration.SubscriptionId);
                 subscriptionIds = (await azureClient.Subscriptions.ListAsync()).Select(s => s.SubscriptionId);
                 resourceManagerClient = GetResourceManagerClient(azureCredentials);
-                MySqlServerName = configuration.ProvisionMySQLOnAzure.GetValueOrDefault() ? SdkContext.RandomResourceName($"{configuration.MainIdentifierPrefix}-", 15) : String.Empty;
+                MySqlServerName = configuration.ProvisionMySqlOnAzure.GetValueOrDefault() ? SdkContext.RandomResourceName($"{configuration.MainIdentifierPrefix}-", 15) : String.Empty;
                 MySqlServerPassword = Utility.GeneratePassword();
+                MySqlSubnetName = configuration.ProvisionMySqlOnAzure.GetValueOrDefault() ? "mysqlsubnet" : String.Empty;
```

Comment:
MySqlSubnetName should be in the Configuration class (as it can be passed in by the user). To reduce clutter, MySqlServerName and MySqlServerPassword should also be part of the Configuration class, but not settable by the user (at least not at this time). 
I suggest changing Configuration.cs as follows, so all the options are in one place:

```
public class Configuration : UserAccessibleConfiguration
{
    public string MySqlServerName { get; set; }
    public string MySqlDatabaseName { get; set; } = "cromwell_db";
    public string MySqlAdministratorLogin { get; set; } = "cromwell";
    public string MySqlAdministratorLoginPassword { get; set; }
    public string MySqlSkuName { get; set; } = "Standard_B1s";
    public string MySqlTier { get; set; } = "Burstable";
    public string DefaultVmSubnetName { get; set; } = "vmsubnet";
    public string DefaultMySqlSubnetName { get; set; } = "mysqlsubnet";
    public string MySqlVersion { get; set; } = "8.0.21";
}

public abstract class UserAccessibleConfiguration   // ***** was Configuration
{
    public string SubscriptionId { get; set; }
    public string RegionName { get; set; }
    public string MainIdentifierPrefix { get; set; } = "abendisdel";
    public string VnetResourceGroupName { get; set; }
    public string VnetName { get; set; }
    public string VmSubnetName { get; set; }
    public string MySqlSubnetName { get; set; }
    public string VnetAddressSpace { get; set; } = "10.1.0.0/16";
    public string VmSubnetAddressSpace { get; set; } = "10.1.0.0/24";
    public string MySqlSubnetAddressSpace { get; set; } = "10.1.1.0/24";
    public string VmName { get; set; }
    public string VmOsProvider { get; set; } = "Canonical";
    ... etc ...
```
and the line 
var configurationProperties = typeof(Configuration)
becomes:
var configurationProperties = typeof(UserAccessibleConfiguration)

Reply:
fixed in latest commit